README.md

```diff
@@ -190,9 +451,15 @@ data = {
     }]
 };
 
-// Insert data. The insert function returns a Promise.
+// The insert function inserts data into the vault and returns a promise.
 const response = client.insert(data, {
-    tokens: true  // Indicates whether or not tokens should be returned for the inserted data. Defaults to "true".
+    tokens: true  // A boolean that indicates if tokens should be returned for the inserted data. The default value is true.
+    upsert:[ // An array that defines support for upsert operations in the vault.
+        {
+            table: "string", // A table name.
+            column: "value", // A unique column in the table.
+        }
+    ]
```

Comment:
Please check this once, it should be table: "<TABLE_NAME>" to maintain consistency across SDK

Reply:
changes done